tinytga/src/header.rs

```diff
@@ -70,16 +101,11 @@ pub struct TgaHeader {
     /// Pixel bit depth (8, 16, 24, 32 bits)
     pub pixel_depth: u8,
 
-    /// Image descriptor (unused)
-    ///
-    /// Bits 0:3: Number of bits per pixel designated to alpha channel
-    /// Bits 4:5: Image origin:
-    ///
-    /// * `00` = bottom left
-    /// * `01` = bottom right
-    /// * `10` = top left
-    /// * `11` = top right
-    pub image_descriptor: u8,
+    /// Image origin
+    pub image_origin: ImageOrigin,
+
+    /// Alpha channel bits
+    pub alpha_channel_bits: u8,
```

Comment:
Would `alpha_channel_depth` be a better name?

Reply:
"Alpha Channel Bits" is used in the TGA specs, but I agree that `alpha_channel_depth` is better and have changed it.